pkg/graph/resolvers/plan_data_exchange_approach.go

```diff
@@ -0,0 +1,209 @@
+package resolvers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+
+	"github.com/cms-enterprise/mint-app/pkg/sqlutils"
+
+	"github.com/cms-enterprise/mint-app/pkg/notifications"
+
+	"github.com/cms-enterprise/mint-app/pkg/email"
+	"github.com/cms-enterprise/mint-app/pkg/shared/oddmail"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cms-enterprise/mint-app/pkg/authentication"
+	"github.com/cms-enterprise/mint-app/pkg/storage/loaders"
+
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/storage"
+)
+
+// PlanDataExchangeApproachGetByID retrieves a plan data exchange approach by its ID
+func PlanDataExchangeApproachGetByID(logger *zap.Logger, store *storage.Store, id uuid.UUID) (*models.PlanDataExchangeApproach, error) {
+	return storage.PlanDataExchangeApproachGetByID(store, logger, id)
+}
+
+// PlanDataExchangeApproachGetByModelPlanID retrieves a plan data exchange approach by its model plan ID
+func PlanDataExchangeApproachGetByModelPlanID(logger *zap.Logger, store *storage.Store, modelPlanID uuid.UUID) (*models.PlanDataExchangeApproach, error) {
```

Comment:
should this be removed in favor of the loader?

Reply:
Probably!
Looks like we only use this function in the [test itself](https://github.com/CMS-Enterprise/mint-app/blob/3d483685f95b4e27d35d85867bd86c9cf9debb57/pkg/graph/resolvers/plan_data_exchange_approach_test.go#L45-L61), which already has a [test for the loader version of the function](https://github.com/CMS-Enterprise/mint-app/blob/3d483685f95b4e27d35d85867bd86c9cf9debb57/pkg/graph/resolvers/plan_data_exchange_approach_test.go#L11-L43)

Reply:
4df5b069a